discovery/cmd/cmd.go

```diff
@@ -33,10 +33,12 @@ func FlagSet() *pflag.FlagSet {
 	flagSet.StringSlice("discovery.server.ids", defs.Server.IDs,
 		"IDs of the Discovery Service for which to act as server. "+
 			"If an ID does not map to a loaded service definition, the node will fail to start.")
-	flagSet.Duration("discovery.client.refresh_interval", defs.Client.RefreshInterval,
+	flagSet.Duration("discovery.client.refreshinterval", defs.Client.RefreshInterval,
 		"Interval at which the client synchronizes with the Discovery Server; "+
 			"refreshing Verifiable Presentations of local DIDs and loading changes, updating the local copy. "+
 			"It only will actually refresh registrations of local DIDs that about to expire (less than 1/4th of their lifetime left). "+
 			"Specified as Golang duration (e.g. 1m, 1h30m).")
+	flagSet.Duration("discovery.client.refresh_interval", 0,
```

Comment:
previously we used https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkDeprecated for this. It logs a warning if the old flag is used, but is not visible in documentation...

(`make cli-docs` if you change this)